roomserver/api/wrapper.go

```diff
@@ -58,7 +60,7 @@ func SendEventWithState(
 			continue
 		}
 		ires = append(ires, InputRoomEvent{
-			Kind:         KindOutlier,
+			Kind:         kind,
```

Comment:
Why would we ever want this to NOT be KindOutlier? Also docs for this function are now lying

Reply:
I'd expect to see this on `:75` not here.

Reply:
Moving this to `:75` makes `Outbound federation can backfill events... OK` but `Backfilled events whose prev_events are in a different room do not allow cross-room back-pagination... FAIL` still

Reply:
You're right, I managed to edit the wrong line there.